recipe/bld.bat

```diff
@@ -0,0 +1,74 @@
+cd msvc_project
+mkdir build
+cd build
+
+MSBuild.exe ../../libp2p/_msvc/libp2p_simd/libp2p_simd.vcxproj /p:configuration=release /p:platform=x64 /p:PlatformToolset=v142 /maxCpuCount:%NUMBER_OF_PROCESSORS% /p:SolutionDir=%cd%\
+if %ERRORLEVEL% neq 0 exit 1
+MSBuild.exe ../Core/Core.vcxproj /p:configuration=release /p:platform=x64 /p:PlatformToolset=v142 /maxCpuCount:%NUMBER_OF_PROCESSORS% /p:SolutionDir=%cd%\
+if %ERRORLEVEL% neq 0 exit 1
+MSBuild.exe ../VSScript/VSScript.vcxproj /p:configuration=release /p:platform=x64 /p:PlatformToolset=v142 /maxCpuCount:%NUMBER_OF_PROCESSORS% /p:SolutionDir=%cd%\
+if %ERRORLEVEL% neq 0 exit 1
+MSBuild.exe ../VSPipe/VSPipe.vcxproj /p:configuration=release /p:platform=x64 /p:PlatformToolset=v142 /maxCpuCount:%NUMBER_OF_PROCESSORS% /p:SolutionDir=%cd%\
+if %ERRORLEVEL% neq 0 exit 1
+MSBuild.exe ../AvsCompat/AvsCompat.vcxproj /p:configuration=release /p:platform=x64 /p:PlatformToolset=v142 /maxCpuCount:%NUMBER_OF_PROCESSORS% /p:SolutionDir=%cd%\
+if %ERRORLEVEL% neq 0 exit 1
+MSBuild.exe ../AVFS/AVFS.vcxproj /p:configuration=release /p:platform=x64 /p:PlatformToolset=v142 /maxCpuCount:%NUMBER_OF_PROCESSORS% /p:SolutionDir=%cd%\
+if %ERRORLEVEL% neq 0 exit 1
+MSBuild.exe ../VSVFW/VSVFW.vcxproj /p:configuration=release /p:platform=x64 /p:PlatformToolset=v142 /maxCpuCount:%NUMBER_OF_PROCESSORS% /p:SolutionDir=%cd%\
+if %ERRORLEVEL% neq 0 exit 1
+
+cd x64/release
+
+copy VapourSynth.dll %PREFIX%\VapourSynth.dll
```

Comment:
This is intended. This script is doing a "portable" install (in VapourSynth's term) so that VapourSynth won't touch Appdata or Registry, which can break environment isolation.
In a "portable" install, the place of `VapourSynth.dll` decides where to search for plugins, and exist VapourSynth users are used to placing plugins in `%PREFIX%/vapoursynth64/plugins`.
Likely in "portable" install `VSScript.dll` tries to load `python%CONDA_PY%.dll` from the same path, so it is placed in `%PREFIX%` as well.

Reply:
> VapourSynth users are used to placing plugins in %PREFIX%/vapoursynth64/plugins

Sorry for not being familiar with the VapourSynth ecosystem, but what does `%PREFIX%` refer to outside conda? Since plugins are usually a bunch of `.dll`s (or `.py`), can't we put the entire `vapoursynth64` inside `%LIBRARY_BIN%`? The plugins dir solely depends on `<VapourSynth.dll path>\vapoursynth64\plugins` as per the [doc](https://www.vapoursynth.com/doc/installation.html).

Anyway, I totally understand that this is just a matter of preference. I just trying to do things the right way.

Reply:
I see. Thank you for your detailed explanation. Indeed, I don't see many use cases for linking VapourSynth though it claims to be a C++ library. We'll get into that when that actually happens.